### ISSUE TITLE
Remote backup fails with WAL file not found?!?!?

### ISSUE THREAD

**Issue by travisas** [pgbackrest/pgbackrest#609]
Please provide the following information when submitting an issue (feature requests or general comments can skip this):

1. pgBackRest version:
pgBackRest 2.05 

2. PostgreSQL version:
Postgres version 11beta

3. Operating system/version - if you have more than one server (for example, a database server, a repository host server, one or more standbys), please specify each:

The repo and db host are the exact same build.   No stand-bys

[postgres@lnx0904 pgbackrest]$ uname -a
Linux lnx0904 3.10.0-327.el7.x86_64 #1 SMP Thu Oct 29 17:29:29 EDT 2015 x86_64 x86_64 x86_64 GNU/Linux
[postgres@lnx0904 pgbackrest]$ cat /etc/system-release
CentOS Linux release 7.2.1511 (Core)

4. Did you install pgBackRest from source or from a package?
Installed from source.

5. Please attach the following as applicable:
 # Add settings for extensions here
archive_command = 'pgbackrest --stanza=test_db archive-push %p'
archive_timeout=1000
archive_mode = on
listen_addresses = '*'
log_line_prefix = ''
max_wal_senders = 3
wal_level = hot_standby

pgbackrest.conf on repo host:
[test_db]
pg1-host=lnx0407.ch3.prod.i.com
pg1-path=/hd/pgramdisk/demo
pg1-port=5432
retention-full=1

[global]
archive-async=y
log-level-file=debug
retention-full=2
retention-diff=1
process-max=10
compress-level-network=5
compress=y
compress-level=5
repo-host=lnx0904.ch3.qa.i.com
repo-path=/pg_backups/test_db
start-fast=y
spool-path=/pgbackrest/spool

[global:archive-get]
process-max=4

[global:archive-push]
process-max=4

7. Describe the issue:

2018-10-09 14:52:03.641 P00  DEBUG:     Storage::Local->list(): bIgnoreMissing = true, strExpression = ^000000010000000000000007-[0-f]{40}(\.gz){0,1}$, strPathExp = <REPO:ARCHIVE>/11-1/0000000100000000, strSortOrder = <forward>
2018-10-09 14:52:03.641 P00  DEBUG:     Storage::Local->list=>: stryFileList = ()
2018-10-09 14:52:03.741 P00  ERROR: [082]: could not find WAL segment 000000010000000000000007 after 60 second(s)
 at /loader/0x1a0ded0/pgBackRest/Archive/Common.pm line 326.
                                        pgBackRest::Archive::Common::walSegmentFind('pgBackRest::Storage::Local=HASH(0x228da48)', '11-1', 000000010000000000000007, 60) called at /loader/0x1a0ded0/pgBackRest/Backup/Backup.pm line 909
                                        pgBackRest::Backup::Backup::process('pgBackRest::Backup::Backup=HASH(0x21da938)') called at /loader/0x1a0ded0/pgBackRest/Main.pm line 263
                                        eval {...} called at /loader/0x1a0ded0/pgBackRest/Main.pm line 286
                                        pgBackRest::Main::main('backup') called at (eval 269) line 1
                                        eval '($iResult, $bErrorC, $strMessage) = pgBackRest::Main::main(\'backup\')
                                   ;' called at (eval 269) line 0

                                   2018-10-09 14:52:03.745 P00  DEBUG:     common/exit::exitSafe: (result: 0, error: true, signalType: 0)
2018-10-09 14:52:03.745 P00  DEBUG:     common/lock::lockRelease: (failOnNoLock: false)
2018-10-09 14:52:03.745 P00  DEBUG:     common/lock::lockRelease: => true
2018-10-09 14:52:03.746 P00  DEBUG:     Main::mainCleanup(): iExitCode = 82
2018-10-09 14:52:03.746 P00  DEBUG:     Protocol::Helper::protocolDestroy(): bComplete = false, iRemoteIdx = [undef], strRemoteType = [undef]
2018-10-09 14:52:03.746 P00  DEBUG:     Protocol::Helper::protocolDestroy=>: iExitStatus = 0
2018-10-09 14:52:03.746 P00   INFO: backup command end: aborted with exception [082]
2018-10-09 14:52:03.746 P00  DEBUG:     common/exit::exitSafe: => 82
2018-10-09 14:52:03.746 P00  DEBUG:     main::main: => 82

**Comment by dwsteele** [comment]
I'm guessing you found the error in the postgres log?

BTW -- 2.05 will not work with 11beta4 because the catalog version changed.  The update is in master but you'll need to build from source if you want to use beta4.  We'll be releasing 2.06 about the same time as the final 11 release.

**Comment by travisas** [comment]
Hi David

That is correct.  I built from source and I successfully backed up the
11beta DB.  I am testing a restore.

Our production systems are PG 10.  I will be building from source in those
environments.


Are there any concerns or things I should read regarding PG 11.  We plan to
upgrade our systems immediately when they release their first stable
version.  Aside from the metadata change.


Thank you
Travis
